library/src/main/java/com/novoda/downloadmanager/lib/DownloadBatch.java

```diff
@@ -50,6 +50,35 @@ public boolean isDeleted() {
         return this == DELETED;
     }
 
+    /**
+     * Return time when this download will be ready for its next action, in
```

Comment:
As far as I could see the name of `action` is a little bit misleading, this returns the time at which an alarm will restart the `DownloadService`.